### PR TITLE
Issue-193

### LINKED ISSUE #193: Tabular view for plots
There should be some form of table view of the data associated with any 2D plot.

### PR DESCRIPTION
**Pull request by emartinez-usgs**
Fixes #193 

This seems a bit complicated over a simple link, but since the service changes based on edition and the dependencies are not resolved within the curve output view itself, it seemed like this approach was less intrusive.

### CODE REVIEW

**Comment by jmfee-usgs** [comment]
I would store the URL on the output object, and set the link target when the data is displayed; rather than recomputing the link target when the link is clicked...  That way users can copy the URL instead of needing to download.

**Comment by emartinez-usgs** [comment]
Updated per @jmfee-usgs suggestions.

File: src/htdocs/js/Calculator.js [hunk 1]
```diff
@@ -68,17 +68,18 @@ var Calculator = function (/*params*/) {
...
+        _this.onXhrSuccess(url, response, service, analysis, callback);
       },
       error: function (err) {
         throw err;
       }
     });
   };
 
-  _this.onXhrSuccess = function (response, service, analysis, callback) {
+  _this.onXhrSuccess = function (url, response, service, analysis, callback) {
```
Comment (ehunter-usgs) [inline src/htdocs/js/Calculator.js]: Need to also change the method signature for `DeaggCalculator.onXhrSuccess`

Comment (emartinez-usgs) [inline src/htdocs/js/Calculator.js]: - Changed signature and also attaching deagg url to model in case we want that in the future too.

**Pull request merged**